docs/section5/Rule5-48.md

```diff
@@ -2,8 +2,8 @@
 # Envelope - Rule 5-48  
 
 **Rule ID:** 5-48  
-**Rule Description:** The air leakage rate, schedule and method in unconditioned and unenclosed spaces must be the same in the baseline and proposed design.  
-**Rule Assertion:** B-RMR infiltration:infiltration.air_leakage_rate, modeling_method, modeling_method_name, multiplier_Schedule= P-RMR infiltration.air_leakage_rate, modeling_method, modeling_method_name, multiplier_Schedule  
+**Rule Description:** The air leakage rate must be the same in the baseline and proposed design.  
+**Rule Assertion:** B-RMR infiltration:air_leakage_rate = P-RMR infiltration:air_leakage_rate  
```

Comment:
Please update per above

Reply:
Updated.

Reply:
The infiltration-related RMR data elements were discussed at the last schema call, and it was agreed to have value for leakage at wind pressure and value for leakage at test pressure (e.g., 75PA). For this RDS we should use the value at wind pressure, whatever the name ends up being. Otherwise it's good to go.

Reply:
Jason will update the schema and xing will update the RDS accordingly.